### DISCUSSION TITLE
[DAI 2024-2025] Ordre de passage pour les présentation du travail pratique 1

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#429] [Announcements]
Bonjour @heig-vd-dai-course/students,

Vous trouverez l'ordre de passage pour les présentations du travail pratique 1 de vos classes respectives à l'adresse suivante, dans le support de cours du travail pratique 1 : <https://github.com/heig-vd-dai-course/heig-vd-dai-course/blob/main/07-practical-work-1/COURSE_MATERIAL.md#presentations>.

Comme annoncé dans la présentation, nous avons seulement 6 minutes par groupe, soit **5 minutes de présentation** de votre part suivi de **1 minute de questions**.

Les présentations auront lieu dans la salle B51a, dans l'angle des escaliers. Un écran sera mis à disposition pour vous permettre de présenter votre travail selon les modalités de la grille d'évaluation. **Merci de venir 5 minutes avant votre créneau horaire avec votre ordinateur**.

Nous serons présents quelques minutes en J01 avec @GeraudSilvestri et @hadrylouis pour vous rappeler la salle et nous irons ensuite en B51a pour vous accueillir et suivre vos présentations.

Merci d'avance et une bonne après-midi,
Ludovic

**Comment by maellemelody** [comment]
Bonjour, est-ce que nous devons tous venir à 8h30 ou peut-on arriver plus tard sur le campus ?
Et qu'y a-t-il de prévu pour la seconde partie de la matinée ?

**Reply by ludelafo** [reply]
Bonjour @Dylan-Langumier,

Vous pouvez venir quand vous le souhaitez, tant que vous êtes là pour votre heure de passage.

En ce qui concerne la seconde partie du cours, nous allons faire la conclusion de la partie 1 du cours et commencer la partie 2 avec son premier chapitre, comme décrit dans le planning ici : https://github.com/orgs/heig-vd-dai-course/projects/2.

Je reste à votre disposition si besoin.

Une bonne journée !
Ludovic Delafontaine

**Reply by maellemelody** [reply]
Merci pour ces clarifications !

Une bonne journée à vous aussi également !
Dylan Langumier
